### ISSUE TITLE
Requesting ipad

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#18496]
> What application are you using?

Other / Not sure

> Describe the problem.

requesting ipad to be given and imaged for Ignacio A. to be able to complete work orders

> How soon do you need this?

Soon — Within a week 

> Is there anything else we should know?

as soon as possible

> Requested By
Jorge L.

Request ID: DTS24-116595

**Comment by scottreedy** [comment]
@Vaishnav1397 Can you take this ticket?

**Comment by Vaishnav1397** [comment]
I checked with matt on the availability of iPad, and he mentioned he would deploy it to the user directly. Adding him on to the ticket @Matthew-M-TPW Can you please update once the deployment is done.

**Comment by Matthew-M-TPW** [comment]
Sent request to CTM to have device moved to our OU in Airwatch

**Comment by Matthew-M-TPW** [comment]
duplicate ticket

**Comment by Matthew-M-TPW** [comment]
https://github.com/cityofaustin/atd-data-tech/issues/18497

**Comment by Matthew-M-TPW** [comment]
Closing ticket due to it being a duplicate of https://github.com/cityofaustin/atd-data-tech/issues/18497
